src/Features/LanguageServer/Protocol/Handler/RequestContext.cs

```diff
@@ -98,27 +104,15 @@ namespace Microsoft.CodeAnalysis.LanguageServer.Handler
                 {
                     var document = documents.FindDocumentInProjectContext(textDocument);
                     logger.TraceInformation($"Found document in workspace {workspace.Kind}: {document.FilePath}");
-
-                    Logger.Log(FunctionId.FindDocumentInWorkspace, KeyValueLogMessage.Create(LogType.Trace, m =>
-                    {
-                        m["WorkspaceKind"] = workspace.Kind;
```

Comment:
we no longer report per request what the workspace was, only the aggregate number of requests that found a document per workspace.  We still have loghub logging though to track per request what the result was.